apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.tsx

```diff
@@ -710,24 +695,27 @@ type OperationDetailsExtraProps = {
   operation: Operation;
   account: Account;
   type: OperationType;
-  extra: {
-    [key: string]: string;
-  };
 };
-const OperationDetailsExtra = ({ extra }: OperationDetailsExtraProps) => {
-  const jsx = Object.entries(extra).map(([key, value]) => {
-    if (typeof value === "object" || typeof value === "function") return null;
-    return (
-      <OpDetailsSection key={key}>
-        <OpDetailsTitle>
-          <Trans i18nKey={`operationDetails.extra.${key}`} defaults={key} />
-        </OpDetailsTitle>
-        <OpDetailsData>
-          <Ellipsis>{value}</Ellipsis>
-        </OpDetailsData>
-      </OpDetailsSection>
-    );
-  });
+const OperationDetailsExtra = ({ operation }: OperationDetailsExtraProps) => {
+  let jsx = null;
+
+  // Safety type checks
+  if (operation.extra && typeof operation.extra === "object" && !Array.isArray(operation.extra)) {
```

Comment:
operation.extra is defined as `Record<string, any>` so it will always be an object, only the value is unknown

Reply:
Not anymore, with this PR the default `Operation.extra` is now of type `unknown` (see `libs/ledgerjs/packages/types-live/src/operation.ts`)